<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastMan</class>
 <widget class="QWidget" name="ContrastMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contrast and W&amp;L</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/resources/gimp-pro/images/gimp-histogram.png</normaloff>:/resources/gimp-pro/images/gimp-histogram.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Linear Contrast Adjustment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_level"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ma&amp;ximum</string>
        </property>
        <property name="buddy">
         <cstring>le_max</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Wi&amp;ndow</string>
        </property>
        <property name="buddy">
         <cstring>le_win</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_min"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="le_win"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_max"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Minimum</string>
        </property>
        <property name="buddy">
         <cstring>le_min</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Le&amp;vel</string>
        </property>
        <property name="buddy">
         <cstring>le_level</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="btn_reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QToolButton" name="btn_auto">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>482</width>
       <height>320</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>482</width>
       <height>320</height>
      </size>
     </property>
     <property name="title">
      <string>Curve based Contrast Adjustment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Histogram options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Bin Size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Cut off:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="value">
         <double>40.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
